### PR TITLE
bluetooth: fast_pair: fp_crypto: Replace double operation with integer

### PR DESCRIPTION
**Pull request by dchat-nordic**
Replace the only double operation in whole fp crypto with an integer operation in the fp_crypto_account_key_filter_size function.

Jira: NCSDK-41386

### CODE REVIEW

File: subsys/bluetooth/fast_pair/fp_crypto/fp_crypto_common.c [hunk 1]
```diff
@@ -89,7 +89,7 @@ size_t fp_crypto_account_key_filter_size(size_t n)
 	if (n == 0) {
 		return 0;
 	} else {
-		return 1.2 * n + 3;
+		return (n * 6) / 5 + 3;
```
Comment (mkapala-nordic) [inline subsys/bluetooth/fast_pair/fp_crypto/fp_crypto_common.c:92]: consider commenting why * 6 / 5 just to easier compare with spec (https://developers.google.com/nearby/fast-pair/specifications/service/provider#AccountKeyFilter)

**Review by mkapala-nordic — APPROVED** [review]

**Review by bartsil — APPROVED** [review]

File: subsys/bluetooth/fast_pair/fp_crypto/fp_crypto_common.c (hunk 1 above)
Comment (kapi-no) [reply subsys/bluetooth/fast_pair/fp_crypto/fp_crypto_common.c:92]: e.g.

```suggestion
	/* Account Key Filter size is defined by the Fast Pair specification as
	 * (1.2 * n + 3) truncated. The 6/5 integer form is equivalent and avoids
	 * pulling in floating-point arithmetic.
	 */
	return (n * 6) / 5 + 3;
```

File: subsys/bluetooth/fast_pair/fp_crypto/fp_crypto_common.c [hunk 2]
```diff
@@ -89,7 +89,7 @@ size_t fp_crypto_account_key_filter_size(size_t n)
...
 	}
 }
```
Comment (kapi-no) [inline subsys/bluetooth/fast_pair/fp_crypto/fp_crypto_common.c:95]: Consider adding ZTEST(suite_crypto, test_bloom_filter_sizes):

tests/subsys/bluetooth/fast_pair/crypto/src/main.c only exercises n = 1 and n = 2 (via first_bloom_filter / second_bloom_filter). Neither is a multiple of 5, so the unit tests would not have caught a rewrite that got the integer-boundary cases wrong. A small table-driven assertion over n = 0 … 10 against the expected sizes above would lock the formula down cheaply.

Comment (dchat-nordic) [reply subsys/bluetooth/fast_pair/fp_crypto/fp_crypto_common.c:95]: I can add it, I tested locally to 2^32 and they match.

Comment (kapi-no) [reply subsys/bluetooth/fast_pair/fp_crypto/fp_crypto_common.c:95]: 2^32 could not possibly work due to the overflow, but this is fine from a practical standpoint.

_n * 6 wraps a 32-bit size_t above n = 715 827 882, where the double version would not. Unreachable in practice — the only caller is fp_advertising.c, which passes the stored account key count bounded by CONFIG_BT_FAST_PAIR_STORAGE_ACCOUNT_KEY_MAX (max 10). Worth knowing, not worth guarding._
